README.md

```diff
@@ -118,7 +118,7 @@ different configurations easily:
 
```

Comment:
Chris - there is a limitations label - has this section gone away?

Reply:
Do you mean the "Limitations - OS compatibility, etc." link in the TOC?  Apparently it went away a long time ago.  

I don't want to creep into fixing too many _content_ errors in this PR (I have a much bigger patch coming up with those kinds of fixes)―this PR is more concerned with fixing doc formatting that has been broken due to actual syntax errors.

Reply:
can we just delete the line in the ToC?
yes that one - otherwise it is a dead link